pkgs/development/tools/misc/lsof/default.nix

```diff
@@ -37,6 +35,10 @@ stdenv.mkDerivation rec {
     for filepath in $(find dialects/${dialect} -type f); do
       sed -i "s,/usr/include,$LSOF_INCLUDE,g" $filepath
     done
+
+    # Wipe out development-only flags from CFLAGS embedding
+    make version.h
+    sed -e "s|$NIX_STORE/[a-z0-9]\{32\}-|$NIX_STORE/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-|g" -i version.h
```

Comment:
Use nukeReferences / nuke-refs instead of sed?

Reply:
Good idea! I also noticed that initial version did not work as `version.h` gets rebuilt on each `make` run. Added a workaround for it to remove phony depend as `sed -i lib/dialects/*/Makefile -e 's/version.h:\s*FRC/version.h:/'`.